iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/operator/OperatorContext.java

```diff
@@ -51,6 +53,10 @@ public class OperatorContext {
 
   private long totalExecutionTimeInNanos = 0L;
   private long nextCalledCount = 0L;
+  private long hasNextCalledCount = 0L;
+  private Map<String, String> specifiedInfo = null;
```

Comment:
add some comments about this field to describe what this field is used for? like what the key and value mean? when this field will be used.